keps/sig-node/5419-pod-level-resources-in-place-resize/README.md

```diff
@@ -102,7 +104,24 @@ This proposal aims to:
 1. Extend the In-Place Pod Resize (IPPR) functionality to support dynamic
    adjustments of pod-level CPU and Memory resources.
 2. Ensure compatibility and proper interaction between pod-level IPPR and existing container-level IPPR mechanisms.
-3. Provide clear mechanisms for tracking and reporting the actual allocated pod-level resources in PodStatus
+3. Provide clear mechanisms for tracking and reporting the actual allocated
+   pod-level resources in PodStatus
+
+### Non Goals
+
+1. This KEP focuses solely on in-place resizing of core compute resources (CPU and
+  Memory) at the pod level. Extending this functionality to other resource types
+  (e.g., GPUs, network bandwidth) is outside the current scope.
+
+2. This KEP does not aim to implement dynamic changes to a pod's QoS class based on
+   in-place resource resize operations. 
+
+3. No dynamic adjustments for Init Containers that have already finished and can't
+    be restarted.
+
+4. No automatic removal of lower-priority pods to make room for a pod that's resizing its resources.
+
+5. This KEP doesn't aim to fix every complex timing issue that can happen between the Kubelet and the scheduler during resizes.
```

Comment:
Can we add a note that this KEP is not fixing timing issues that already exist in other cases? Otherwise this statement is very broad =).